### ISSUE TITLE
Error while loading shared libraries

### ISSUE THREAD

**Issue by JimiPedros** [electron/electron-quick-start#519]
Ubuntu 20.04.2 LTS

Steps to reproduce:
- clone electron quick start
- npm i 
- npm run start

Result:
```
/node_modules/electron/dist/electron: error while loading shared libraries: libatk-1.0.so.0: cannot open shared object file: No such file or directory
error Command failed with exit code 127.
```

**Comment by genru** [comment]
if you are running on wsl system. here would help https://github.com/electron/electron-quick-start/issues/486#issuecomment-1050782211

**Comment by giuliohome** [comment]
> if you are running on wsl system. here would help [#486 (comment)](https://github.com/electron/electron-quick-start/issues/486#issuecomment-1050782211)

Nope, look at [my comment there instead for WSL](https://github.com/electron/electron-quick-start/issues/486#issuecomment-1264321797)

**Comment by ackvf** [comment]
> Nope, look at [my comment there instead for WSL](https://github.com/electron/electron-quick-start/issues/486#issuecomment-1264321797)

I confirm that installing chrome under WSL works out of the box and might be a better cross-platform solution than combining WSL with `platform=win32`, which, as others reported, might not even work for some.

```
sudo apt update
wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
sudo dpkg -i google-chrome-stable_current_amd64.deb
sudo apt-get install -f
google-chrome --version
```

If you get issues along the way, follow the link in the original post above.
